src/Parser/FrostMage/Modules/Items/ShardOfTheExodar.js

```diff
@@ -0,0 +1,43 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import { formatNumber } from 'common/format';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class ShardOfTheExodar extends Module {
+
+	damage = 0;
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasFinger(ITEMS.SHARD_OF_THE_EXODAR.id);
+  }
+
+  on_byPlayer_damage(event){
+    const BLOODLUST_BUFFS = [
```

Comment:
This array is constant and we usually extract constants out _before_ the class even begins. Just move it, no other changes needed.